src/site-specific-feature.js

```diff
@@ -0,0 +1,46 @@
+import ConfigFeature from '@duckduckgo/content-scope-scripts/injected/src/config-feature';
+
+const FEATURE_NAME = 'site-specific-fixes';
+
+export default class SiteSpecificFeature extends ConfigFeature {
+    constructor(args) {
+        super(FEATURE_NAME, args);
+    }
+
+    /**
+     * @returns {import('@duckduckgo/privacy-configuration/schema/features/autofill.js').SiteSpecificFixes['formTypeSettings']}
+     */
+    get formTypeSettings() {
+        return this.getFeatureSetting('formTypeSettings') ?? [];
```

Comment:
You're very fond of the `??` operator, aren't you? 🙂 TBH, unless it's really needed for specific reasons (i.e. when your value can be `0` or `''`) I prefer the good old `||`. This seems to imply a specific need that it's not there (i.e. as if we were expecting this to be either `0` or `''` which we don't).